inc/vcf/record_cache.hpp

```diff
@@ -113,8 +140,26 @@ namespace ebi
             }
         }
 
+        /**
+         *Checks if the alternate allele is symbolic 
+         */ 
+        bool is_symbolic_allele(const RecordCore &record_core)
```

Comment:
If you modify the `RecordCore` class to have the variant type as an attribute, this method would be no longer needed.